.github/workflows/go-lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: Code Linting
+
+on:
+  push:
+    branches: ["main"]
+
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  lint:
+    name: Code Linting
+    strategy:
+      matrix:
+        go-version: ["1.18"]
+        platform: ["ubuntu-latest"]
+    runs-on: ${{ matrix.platform }}
+    timeout-minutes: 5
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

Comment:
What's this opaque string?  Just curious.

Reply:
best practice to use commit SHAs instead of branch names or tags to prevent abuse

Reply:
Fair enough. [Googling](https://www.google.com/search?client=safari&rls=en&sxsrf=ALiCzsZCexC7UB6Y_ktzIAvWIIr4FO5YTg:1653493427591&q=actions/checkout@+%222541b1294d2704b0964813337f33b291d3f8596b%22&sa=X&ved=2ahUKEwixi4-D__r3AhVKiI4IHSgDABgQ5t4CegQIGxAB&biw=1725&bih=1015&dpr=2) reveals this is used a lot of places.  Pardon me but I'm curious, is there a way to use this as a link to see what is actually being referenced?